server/models.ts

```diff
@@ -19,7 +19,7 @@ export const sequelize = new Sequelize(process.env.DATABASE_URL, {
 	},
 });
 
-export const knex = knexJs({ client: 'pg' });
+export const knexInstance = knex({ client: 'pg' });
```

Comment:
Based on [the docs](https://knexjs.org/guide/#configuration-options) I think it's idiomatic when working with this library to call the instance object `knex`. Their examples use CommonJS requires in a way that we don't want to replicate:

```ts
const knex = require('knex')({
  client: 'mysql',
  ...
});
```

But the fact that they don't even assign a symbol to `require('knex')` suggests that we shouldn't worry about "naming it properly" or disambiguating it from the instances that it creates. So I think we could `import { knex as createKnex }` or sth and get the `knex` name back for the code that uses it.

Reply:
I see your point, and I actually considered aliasing the `knex` import but this naming is actually from their docs too: https://github.com/knex/knex/blob/master/UPGRADING.md#upgrading-to-version-0950. Personally I felt like this was less likely to lead to someone accidentally importing the wrong `knex` in the future, so I'm going to leave it as is for now. But if someone in the future sees this comment and disagrees, feel free to change it!